### DISCUSSION TITLE
Ask Autorenewal

### DISCUSSION THREAD

**Discussion by Ekayul** [community/community#131075] [Enterprise]
### Select Topic Area

Question

### Body

Hi, we are planning to buy Github Enterprise

Please let me know, does Github Enterpise have autorenewal?
If yes, how do i turn it off?

**Comment by queenofcorgis** [comment]
Hi there, 

You have a few options for managing your billing cycle, this doc may help: https://docs.github.com/en/billing/managing-your-github-billing-settings/changing-the-duration-of-your-billing-cycle

**Reply by Ekayul** [reply]
Hi, i have seen the article you sent but this is not to turn off autorenewal, but to change the billing cycle from monthly to annual or vice versa.
So how do we turn off autorenewal?

**Reply by queenofcorgis** [reply]
Hi @Ekayul - thank you for your patience. If you pay by card or PayPal you will not be able turn off auto renewal. If you plan on purchasing at least 10 seats on an annual basis, you can opt for the invoice payment method. [This doc](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-the-plan-for-your-github-account/managing-invoices-for-your-enterprise) shares a bit more about the details of this process.

**Reply by Ekayul** [reply]
Hi, if next year we don't want to use GitHub anymore, how do we stop the subscription from being renewed?

Because of your previous information that we can't turn off autorenewal.

**Reply by queenofcorgis** [reply]
@Ekayul would you mind opening a ticket for this, on our [Support page](https://support.github.com/contact) please? Feel free to share the ticket number with me so I can brief the team on what we've discussed already 😄

**Comment by Ekayul** [comment]
Hi Lili

Yes, i have opened a ticket, here is my ticket number 2860860

Dapatkan Outlook untuk Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by queenofcorgis** [reply]
Thanks for opening the ticket and sharing the number with me, I passed along the details of this Discussion to the team.

**Comment by Ekayul** [comment]
Hi Lili

Thank you, FYI we will purchase github enterprise for 1 qty for this account ***@***.******@***.***>


Regards,
Eka Yulianti

**Reply by queenofcorgis** [reply]
@Ekayul - wonderful to hear! For any more questions or feedback, we're here for you in the community!

**Reply by Ekayul** [reply]
Hi, 
Any update regarding this case?
Is there another solution to this problem so that next year it doesn't renew automatically?
because wa want to purchase that item soon.
